src/Homework2/Month.java

```diff
@@ -0,0 +1,51 @@
+package Homework2;
+
+public class Month {
+
+    public void month (int monthNumber){
```

Comment:
you should be create array with name of month and get them by index, too long case is not good idea